make.js

```diff
@@ -402,6 +239,195 @@ CLI.serverBuild = function() {
     banner('Build successful', true);
 }
 
+function buildTasks(taskName) {
+      let isGeneratedTask = false;
+      banner('Building: ' + taskName);
+      const removeNodeModules = taskList.length > 1;
```

Comment:
>removeNodeModules [](http://example.com/codeflow?start=12&length=17)

I think we need to check the filtered list for this, not "taskList", maybe pass it in as an arg?